components/abtesting/optimizelyXExperiment/src/optimizely-x.js

```diff
@@ -0,0 +1,65 @@
+import ExperimentUseCase from './use-case'
+
+class OptimizelyXExperimentsService {
+  /**
+   * Get OptimizelyX SDK instance
+   * @return Promise<Object>
+   */
+  static async getSDK () {
+    return window && window.optimizely
+  }
+
+  /**
+   * Get info of an experiment
+   * @param {String} experimentId
+   * @return Promise<Object>
+   * @example
+   * {
+   *  id: '100',
+   *  isActive: true,
+   *  isInExperimentHoldback: true,
+   *  variation: {
+   *      id: '101',
+   *      name: 'Variation #1'
+   *  }
+   * }
+   */
+  static async getInfo (experimentId) {
+    let sdk = await this.getSDK()
```

Comment:
BTW I changed it [here](https://github.com/SUI-Components/sui-components/commit/71b6b4a21b1250ca80fa04785577b765f9187ca3#diff-c271d8346f679e021e3b8f11a422901dL54). I don't have a strong opinion about this and I get that we're defining an immutable reference to an object. Maybe we should talk altogether and decide something as a convention. Actually I don't event like to name consts without uppercase. Maybe I'm just old school.